_posts/2018-05-09-Stop-_-Frisk.md

```diff
@@ -234,26 +234,26 @@ Building off the "crime-only" model, now that we have access to demographic crim
 </table>
 
 
-There are plenty of great (better*) resources on the internet/in the public domain that can explain the details of Poisson regressions. However, if you're like me and just want to skim the relevant Cross-Validated/Stack Overflow posts to know the gist of how the model should be interpreted, then here's a quick explanation of the results above: The parameters displayed represent the multiplicative increase in stop and frisk for all neighborhood race compositions compared to the specific base factor. In the case of race, the base is white and in the case of neighborhood percent black, it's 0% - 10% black.
+There are plenty of great (better*) resources on the internet/in the public domain that can explain the details of Poisson regressions. However, if you're like us and just want to skim the relevant Cross-Validated/Stack Overflow posts to know the gist of how the model should be interpreted, then here's a quick explanation of the results above: The parameters displayed represent the multiplicative increase in stop and frisk for all neighborhood race compositions compared to the specific base factor. In the case of race, the base is white and in the case of neighborhood percent black, it's 0% - 10% black. THIS SENTENCE SEEMS GRAMMATICALLY QUESTIONABLE BUT I DONT UNDERSTAND IT SO I CANT FIX IT. 
```

Comment:
GW: yeah, this does look/sound awkward. I edited it up a bit. Can you look in the main drafts branch and let me know if it's any better?